Makefile

```diff
@@ -6,7 +6,7 @@ TEST_LOCATION	= ./...
 TEST_CMD		= go test -v -mod=vendor -cover ${TEST_LOCATION}
 TEST_CMD_COV	= ${TEST_CMD} -covermode=count -coverprofile=coverage.out
 
-VERSION			= v1.0.0
+VERSION			= v1.0.1
```

Comment:
As I said about get the latest version, WDYT?
```
VERSION = `git describe --tags $(git rev-list --tags --max-count=1)`
```

Reply:
that's looks good
however... I'm having 2nd feelings, this seems weird: if the latest tag is 5 commits ago, are you really running that version of µTask?

Reply:
It's the problem of the end user I think.
It always recommended to fetch the latest release instead of clone the master branch.
And if it's a tech dev want to fork or anything on his side, he can also do a `git log` to see the history upstream.

Reply:
fair enough, I'll apply your proposal

Reply:
👯‍♂